lib/ansible/modules/cloud/azure/azure_rm_virtualmachine.py

```diff
@@ -1472,17 +1474,19 @@ def exec_module(self, **kwargs):
 
         return self.results
 
-    def get_vm(self):
+    def get_vm(self, fail_on_exception=True):
```

Comment:
why need the fail_on_exception? seems all callers pass false

Reply:
It used to pass True, here I just add one False caller...